chains/substrate/listener.go

```diff
@@ -76,10 +76,10 @@ func (l *listener) start() error {
 	}
 
 	go func() {
-		err := l.pollBlocks()
-		if err != nil {
-			l.log.Error("Polling blocks failed", "err", err)
-		}
+		// err := l.pollBlocks()
```

Comment:
We should add a comment on why this part has been commented on.

Reply:
We should try to disable scanning earlier. Please, research.

Reply:
Added comment. 
I tried to disable it in the place of connecting to the substrate. But it throws registering a new router error. 
That's why I have disabled it at the place of scanning.

Reply:
> But it throws registering a new router error
Have you researched for what does it mean?

Reply:
During initialization, the system will fetch whitelist id from blockchin. So we can't stop at that point. So I have commented on the block polling point.